bigflow/dataflow.py

```diff
@@ -18,8 +18,9 @@ def __init__(
             pipeline_options: PipelineOptions = None,
             entry_point_arguments: typing.Optional[dict] = None,
             wait_until_finish: bool = True,
-            execution_timeout: int = 3600000,
-            test_pipeline: Pipeline = None
+            execution_timeout: int = DEFAULT_JOB_EXECUTION_TIMEOUT - 120000,  # minus two minutes to allow airflow report to user timeout of BeamJob
```

Comment:
Why there are 2 timeout parameters? (execution_timeout and job_execution_timeout)

Reply:
job_execution_timeout is used in DAG and passed to KubernetesPodOperator. It is used to control timeout at airflow side. execution_timeout is timeout at dataflow